src/transformer/utils/get_user.php

```diff
@@ -19,7 +19,8 @@
 
 function get_user(array $config, \stdClass $user) {
     $fullname = get_full_name($user);
-    $hasvalidemail = filter_var($user->email, FILTER_VALIDATE_EMAIL);
+    // The following email validation matches that in Learning Locker
+    $hasvalidemail = mb_ereg_match("[A-Z0-9\\.\\`\\'_%+-]+@[A-Z0-9.-]+\\.[A-Z]{1,63}$", $user->email, "i");
```

Comment:
The regex here will need a `^` at the beginning to ensure that it matches from the beginning of the string.

Reply:
The [mb_ereg_match](http://php.net/manual/en/function.mb-ereg-match.php) function matches from the beginning of the string automatically.

Reply:
Ah didn't know that, thanks @AndyHubert, seems good then 👍